deployments/local-setup-instance/provisioning/run-install.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -e
+
+cd local-setup
+
+while true; do
+  echo "Checking Geth state..."
+
+  ETH_BLOCK=$(curl -s -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":67}' http://localhost:8545 | jq -r '.result')
+
+  if [ -z "$ETH_BLOCK" ] || [ "$ETH_BLOCK" == "0x0" ]; then
+    echo "Geth still not initialized - waiting..."
+    sleep 30
+    continue;
+  fi
+
+  echo "Geth initialized!"
+
+  break
+done
+
+while true; do
+  echo "Checking Bitcoin Core state..."
+
+  BTC_BLOCK=$(curl -s --user user:password --data-binary '{"jsonrpc": "1.0", "id": "curltest", "method": "getblockcount", "params": []}' -H 'content-type: text/plain;' http://127.0.0.1:18332 | jq -r '.result')
+
+  if [ -z "$BTC_BLOCK" ] || [ "$BTC_BLOCK" == "0" ]; then
+    echo "Bitcoin Core still not initialized - waiting..."
+    sleep 30
+    continue;
+  fi
+
+  echo "Bitcoin Core initialized!"
+
+  break
+done
+
+echo "Running install script..."
+
+./install.sh
```

Comment:
`vagrant up` failed for me with:

```
    (...)
    default: Checking Bitcoin Core state...
    default: Bitcoin Core initialized!
    default: Running install script...
    default:
    default: Initializing submodules...
    default:
    default: Submodule 'keep-core' (https://github.com/keep-network/keep-core.git) registered for path 'keep-core'
    default: Submodule 'keep-ecdsa' (https://github.com/keep-network/keep-ecdsa.git) registered for path 'keep-ecdsa'
    default: Submodule 'relays' (https://github.com/keep-network/relays.git) registered for path 'relays'
    default: Submodule 'tbtc' (https://github.com/keep-network/tbtc.git) registered for path 'tbtc'
    default: Submodule 'tbtc-dapp' (https://github.com/keep-network/tbtc-dapp.git) registered for path 'tbtc-dapp'
    default: Submodule 'tbtc.js' (https://github.com/keep-network/tbtc.js.git) registered for path 'tbtc.js'
    default: Cloning into 'keep-core'...
    default: Submodule path 'keep-core': checked out 'c6af1a5c49860b7f1ba2f015adf820c01aa7b2e7'
    default: Cloning into 'keep-ecdsa'...
    default: Submodule path 'keep-ecdsa': checked out '89bd7bffea7282da3212b9a3f6b4f559725c3e67'
    default: Cloning into 'relays'...
    default: Submodule path 'relays': checked out 'efea3a8590e821c617256d6ef56ada956319c6ba'
    default: Cloning into 'tbtc'...
    default: Submodule path 'tbtc': checked out 'd82b54abe41d6d96c01734c9a5d6926b47447b20'
    default: Cloning into 'tbtc-dapp'...
    default: Submodule path 'tbtc-dapp': checked out 'e00534f812c7e4b54f27f5b8f41bb79f6f7c947f'
    default: Cloning into 'tbtc.js'...
    default: Submodule path 'tbtc.js': checked out '6890d466d1a32793d4834a1e0d35d30c0acd047e'
    default:
    default: Starting keep-core deployment...
    default:
    default: Copying config files...
    default:
    default: sed: -e expression #1, char 1: unknown command: `.'
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

Reply:
I didn't mention this in the description, sorry! This will work after this PR will be merged. This is because `/deployments/local-setup-instance/provisioning/clone-repository.sh` script always clones the recent `master` of `local-setup` and performs all actions against it. Because the current `master` doesn't have some changes which will make everything work, you see this error. As a quick workaround please modify the above script as it will look like:

```
#!/bin/bash

set -e

echo "Cloning repository..."

if [ -d local-setup ]; then rm -Rf local-setup; fi

git clone https://github.com/keep-network/local-setup.git

cd local-setup
git checkout local-setup-vm

echo "Repository has been cloned successfully!”
```

Reply:
OK, after altering `clone-repository.sh` I was able to start the machine. 👍